float64[] offset  # [x, y, z, roll, pitch, yaw]
---
bool success
string message